protocol/grpc/grpc_invoker.go

```diff
@@ -150,3 +167,16 @@ func (gi *GrpcInvoker) Destroy() {
 		}
 	})
 }
+
+// timeout
+func (gi *GrpcInvoker) getTimeout(invocation protocol.Invocation) time.Duration {
+	var timeout time.Duration
+	timeoutStr := gi.BaseInvoker.GetURL().GetParam(constant.METHODS_KEY+"."+invocation.MethodName()+"."+constant.TIMEOUT_KEY,
+		gi.BaseInvoker.GetURL().GetParam(constant.TIMEOUT_KEY, ""))
+	if timeoutStr != "" {
+		timeout, _ = time.ParseDuration(timeoutStr)
+	} else {
+		timeout = gi.timeout
+	}
+	return timeout
```

Comment:
```suggestion
	if len(timeoutStr)==0 {
		timeout = gi.timeout
	} 
	timeout, _ = time.ParseDuration(timeoutStr)
	return timeout
```
what about this style?